coverage.html

```diff
@@ -135,7 +135,7 @@
 				
 				<option value="file39">github.com/Peersyst/xrpl-go/pkg/map_utils/map_utils.go (100.0%)</option>
 				
-				<option value="file40">github.com/Peersyst/xrpl-go/pkg/random/main.go (80.0%)</option>
+				<option value="file40">github.com/Peersyst/xrpl-go/pkg/random/main.go (0.0%)</option>
```

Comment:
Do you know why it dropped to 0 ?

Reply:
That's because of the decoupling between packages. That's an indicator that `random` package should have each own tests, which I'm adding right now